specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2021-01-01/kusto.json

```diff
@@ -0,0 +1,4925 @@
+            "Standard_DS13_v2+2TB_PS",
+            "Standard_DS14_v2+3TB_PS",
+            "Standard_DS14_v2+4TB_PS",
+            "Standard_D13_v2",
+            "Standard_D14_v2",
+            "Standard_L8s",
+            "Standard_L16s",
+            "Standard_L8s_v2",
+            "Standard_L16s_v2",
+            "Standard_D11_v2",
+            "Standard_D12_v2",
+            "Standard_L4s",
+            "Dev(No SLA)_Standard_D11_v2",
+            "Standard_E64i_v3",
+            "Standard_E80ids_v4",
+            "Standard_E2a_v4",
+            "Standard_E4a_v4",
+            "Standard_E8a_v4",
+            "Standard_E16a_v4",
+            "Standard_E8as_v4+1TB_PS",
+            "Standard_E8as_v4+2TB_PS",
+            "Standard_E16as_v4+3TB_PS",
+            "Standard_E16as_v4+4TB_PS",
+            "Dev(No SLA)_Standard_E2a_v4"
+          ],
+          "x-ms-enum": {
+            "name": "AzureSkuName",
+            "modelAsString": true
+          }
+        },
+        "capacity": {
+          "description": "The number of instances of the cluster.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "tier": {
+          "description": "SKU tier.",
+          "type": "string",
+          "enum": [
+            "Basic",
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "AzureSkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Azure SKU definition."
+    },
+    "Zones": {
+      "description": "An array represents the availability zones of the cluster.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "OptimizedAutoscale": {
+      "type": "object",
+      "required": [
+        "version",
+        "isEnabled",
+        "minimum",
+        "maximum"
+      ],
+      "properties": {
+        "version": {
+          "description": "The version of the template defined, for instance 1.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "isEnabled": {
+          "description": "A boolean value that indicate if the optimized autoscale feature is enabled or not.",
+          "type": "boolean"
+        },
+        "minimum": {
+          "description": "Minimum allowed instances count.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "maximum": {
+          "description": "Maximum allowed instances count.",
+          "type": "integer",
+          "format": "int32"
+        }
+      },
+      "description": "A class that contains the optimized auto scale definition."
+    },
+    "VirtualNetworkConfiguration": {
+      "type": "object",
+      "required": [
+        "subnetId",
+        "enginePublicIpId",
+        "dataManagementPublicIpId"
+      ],
+      "properties": {
+        "subnetId": {
+          "description": "The subnet resource id.",
+          "type": "string",
+          "x-ms-mutability": [
```

Comment:
Users can't read these properties after they are set?  Most of the time a proprety that cant be updates is set to "create", "read".   See https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-mutability

Reply:
Fixed, thanks.